Assets/Scripts/HUD/GameResources.cs

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
I'm still seeing a lot of mixed tabs and spaces in this file.

Please convert all tab characters to 4 space characters.